src/redux/migrations.ts

```diff
@@ -1416,4 +1416,5 @@ export const migrations = {
       },
     }
   },
+  166: (state: any) => state,
```

Comment:
This should add the `hideBalances` state to app. You can see other migrations above for examples.